bcrs-ui/src/utils/axios-auth.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios'
```

Comment:
Good decision to move this into utils/ !

Reply:
awesome, glad you like, thanks!